src/qpmodel.jl

```diff
@@ -64,7 +65,8 @@ function QuadraticModel(c :: AbstractVector, H :: AbstractMatrix;
     nnz(A), findnz(A)...
   else
     I = ((i,j,A[i,j]) for i = 1:ncon, j = 1:nvar)
-    nvar * ncon, getindex.(I, 1)[:], getindex.(I, 2)[:], getindex.(I, 3)[:]
+    nvar * ncon, convert(Array{Int, 1}, getindex.(I, 1)[:]),
```

Comment:
Why is this needed?

Reply:
Thank you for your comments.
I don't really know why it is needed, but when A is empty, `getindex.(I, 1)[:]` is `Any[]`. The error message is:

`ERROR: LoadError: LoadError: MethodError: no method matching QuadraticModels.QPData(::Float64, ::Array{Float64,1}, ::Array{Int64,1}, 
::Array{Int64,1}, ::Array{Float64,1}, ::Array{Union{Float64, Int64},1}, ::Array{Union{Float64, Int64},1}, ::Array{Union{Float64, Int64},1})
Closest candidates are:
  QuadraticModels.QPData(::T, ::Array{T,1}, ::Array{Int64,1}, ::Array{Int64,1}, ::Array{T,1} where T, ::Array{Int64,1}, ::Array{Int64,1}, ::Array{T,1} where T) where T at C:\Users\Geoffroy Leconte\.julia\dev\QuadraticModels.jl\src\qpmodel.jl:4`

I am sorry for the github issue, does merging jso/master into this branch will solve it? (I work with git CLI)

Reply:
About the git issue, nope. I created a walkthrough here: https://asciinema.org/a/361816 - The screen was cut, the graph command is `git log --oneline --graph --all --decorate`.

Some guidance for the future:
- Your master is never leading, always following. So after the PR is merged, use `git fetch --all` to update your remotes, `git chekout master` and `git merge jso/master --ff-only` to update your master.
- Go to the master, branch, create a/some commit(s) and create a pull request for that branch
- Repeat as needed, always branching from the master
- If the pull requests are "orthogonal", we can merge them without conflicts, otherwise you're gonna have to rebase (see below)
- If some pull requests conflicted, you're gonna have to rebase your conflicted branch to the new master. Say `issueA` can't be merged and we ask you to rebase. Update the master as above then
  - `git rebase master issueA`
  - Conflicts will arise. Open your editor and fix them.
  - `git add files-that-had-conflict`
  - `git rebase --continue`
  - The larger the commit difference, the larger the conflict.
- When a branch is merged, we use rebase or squash, which means that your branch information is used, but not the actual commit. You can safely delete your branch because the master is now updated.

Reply:
I think it's better to add a `if ncon == 0` special case in line 64:
```
nnzj, Arows, Acols, Avals = if ncon == 0
    0, Int[], Int[], T[]
elseif issparse(A)
    nnz(A), findnz(A)...
```

Reply:
@abelsiqueira It would be great to add such a guide on git branching to our org's wiki.

Reply:
Thank you! I hope it is ok now.